packages/edit-site/src/components/layout/hooks.js

```diff
@@ -5,15 +5,9 @@ import { useEffect, useState } from '@wordpress/element';
 import { useSelect } from '@wordpress/data';
 import { store as coreStore } from '@wordpress/core-data';
 
-/**
- * Internal dependencies
- */
-import useEditedEntityRecord from '../use-edited-entity-record';
-
 const MAX_LOADING_TIME = 10000; // 10 seconds
 
 export function useIsSiteEditorLoading() {
-	const { isLoaded: hasLoadedPost } = useEditedEntityRecord();
```

Comment:
my thinking is that this is a resolver as well, so there's no need to add this specifically here.

Reply:
> This makes sense. I added a conditional breakpoint though and sometimes hasLoadedPost = false and loaded = true.

Was this during the initial loading or when you navigated between the pages? Seems the latter right and in that case it seems that it's probably ok.

Reply:
The latter. Yeah, I think it will be fine.